cli-script.php

```diff
@@ -82,23 +82,20 @@
 // Connect to database
 if(!isset($databaseConfig) || !isset($databaseConfig['database']) || !$databaseConfig['database']) {
 	echo "\nPlease configure your database connection details.  You can do this by creating a file
-called _ss_environment.php in either of the following locations:\n\n";
-	echo " - " .  BASE_PATH  . DIRECTORY_SEPARATOR . "_ss_environment.php\n - ";
-	echo dirname(BASE_PATH) . DIRECTORY_SEPARATOR . "_ss_environment.php\n\n";
+called .env in " . BASE_PATH;
```

Comment:
In addition to this, it would be nice if I could split "server level" and "site level" specific configs. E.g. `~/Sites/.env` has my global settings, but I can override specifics with `~/Sites/mysite/.env`.